source/common/runtime/runtime_features.cc

```diff
@@ -108,6 +108,7 @@ RUNTIME_GUARD(envoy_restart_features_allow_client_socket_creation_failure);
 RUNTIME_GUARD(envoy_restart_features_quic_handle_certs_with_shared_tls_code);
 RUNTIME_GUARD(envoy_restart_features_send_goaway_for_premature_rst_streams);
 RUNTIME_GUARD(envoy_restart_features_udp_read_normalize_addresses);
+RUNTIME_GUARD(envoy_restart_features_use_fast_protobuf_hash);
```

Comment:
I guess there is no UT to check default values, is that common?

Reply:
There are unit tests that test both true and false behaviors for this case - I made the unit tests force the values specifically so that changing the default wouldn't require updating the tests.
It may not be common, but I find it much less annoying than the other way.